### DISCUSSION TITLE
Logging trades

### DISCUSSION THREAD

**Discussion by kosi96** [kernc/backtesting.py#258] [Q&A] [answered]
Hello,

is there some way to log trades that were made? 
I would like to have it in some kind of text format to filter it by worst/best performance..

**Comment by kernc** [comment, marked as answer]
Have you seen [_Trade data_ section in the _Quick Start_ guide](https://kernc.github.io/backtesting.py/doc/examples/Quick%20Start%20User%20Guide.html#Trade-data)?

**Reply by kosi96** [reply]
Missed it, thank you for fast and accurate response!
